power_grid_model_c/power_grid_model/include/power_grid_model/main_core/update.hpp

```diff
@@ -96,22 +101,137 @@ inline void update_inverse(MainModelState<ComponentContainer> const& state, Forw
         begin, end, sequence_idx);
 }
 
+#define INCREMENT_UPDATE_Y_BUS
```

Comment:
What are we doing here with a macro?

Reply:
It's meant to be removed before merging

Reply:
Processed in the new commit, conversation resolved